### PR TITLE
Add parser proc_environ

### PR DESCRIPTION
**Pull request by shzhou12**

### CODE REVIEW

**Comment by bfahr** [comment]
@shzhou12 does this replace #1858?  If so should it be closed, and #1866 merged to revert the parser from #1848?

**Comment by shzhou12** [comment]
@bfahr  Yes, this pr is to replace #1858 . I have closed #1858.

File: insights/specs/__init__.py [hunk 1]
```diff
@@ -371,7 +371,11 @@ class Specs(SpecSet):
     openvswitch_other_config = RegistryPoint()
     openvswitch_server_log = RegistryPoint(filterable=True)
     openshift_certificates = RegistryPoint(multi_output=True)
+    openshift_fluentd_pid = RegistryPoint()
```
Comment (xiangce) [inline insights/specs/__init__.py]: This spec is not necessary here.

File: insights/specs/__init__.py [hunk 2]
```diff
@@ -371,7 +371,11 @@ class Specs(SpecSet):
...
+    openshift_fluentd_environ = RegistryPoint(multi_output=True)
     openshift_hosts = RegistryPoint(filterable=True)
+    openshift_router_pid = RegistryPoint()
```
Comment (xiangce) [inline insights/specs/__init__.py]: Same comment as above

File: insights/specs/insights_archive.py [hunk 3]
```diff
@@ -178,6 +178,8 @@ class InsightsArchiveSpecs(Specs):
     oc_get_route = simple_file("insights_commands/oc_get_route_-o_yaml_--all-namespaces")
     oc_get_service = simple_file("insights_commands/oc_get_service_-o_yaml_--all-namespaces")
     oc_get_configmap = simple_file("insights_commands/oc_get_configmap_-o_yaml_--all-namespaces")
+    openshift_fluentd_pid = simple_file("insights_commands/pgrep_-n_fluentd")
```
Comment (xiangce) [inline insights/specs/insights_archive.py]: For `foreach_collect`, it's not necessary to be added to the `insights_archive.py` for now.

File: insights/specs/insights_archive.py [hunk 4]
```diff
@@ -178,6 +178,8 @@ class InsightsArchiveSpecs(Specs):
...
+    openshift_router_pid = simple_file("insights_commands/pgrep_-n_openshift-route")
```
Comment (xiangce) [inline insights/specs/insights_archive.py]: Same content as above

File: insights/parsers/proc_environ.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
+"""
+ProcEnviron - File ``/proc/<PID>/environ``
+==========================================
+
+Parser for parsing the the ``environ`` file under ``/proc/<PID>``
```
Comment (xiangce) [inline insights/parsers/proc_environ.py]: `the the` -> `the`

**Pull request merged**